man/udevadm.xml

```diff
@@ -382,6 +382,21 @@
             then each matching result is ORed, that is, all children of each specified device are triggered.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>--database-nomatch</option></term>
+          <listitem>
+            <para>Trigger devices without a corresponding udev database entry only. This can potentially save a
```

Comment:
Maybe: "Trigger only those devices that do not yet possess a matching udev database entry."

i.e. "only" matters I think. And "yet" matters.